keyboards/nyquist/keymaps/bramver/keymap.c

```diff
@@ -0,0 +1,286 @@
+ * |------+------+------+------+------+------| |------+------+------+------+------+------|
+ * | Ctrl |      | Alt  | GUI  |      |      | |      |      | GUI  | Alt  |      | Ctrl |
+ * `-----------------------------------------' `-----------------------------------------'
+ */
+
+[_RMOUSE] = LAYOUT( \
+  TO(0)   , XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX , /**/  KC_PGUP ,   KC_PGDN ,   KC_HOME ,   KC_END  ,   XXXXXXX ,   _______ , \
+  _______ , XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX , /**/  KC_WH_U ,   KC_BTN1 ,   KC_MS_U ,   KC_BTN2 ,   KC_WH_L ,   _______ , \
+  XXXXXXX , XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX , /**/  KC_WH_D ,   KC_MS_L ,   KC_MS_D ,   KC_MS_R ,   KC_WH_R ,   KC_DEL  , \
+  _______ , XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX , /**/  XXXXXXX ,   KC_LEFT ,   KC_DOWN ,   KC_UP   ,   KC_RGHT ,   _______ , \
+  _______ , XXXXXXX ,   _______ ,   _______ ,   _______ ,   XXXXXXX , /**/  _______ ,   _______ ,   _______ ,   _______ ,   XXXXXXX ,   _______  \
+),
+
+/* Emojis
+ * ,-----------------------------------------. ,-----------------------------------------.
+ * | GESC |      |      |      |      |      | | RSET |      |      |      |      | Bksp |
+ * |------+------+------+------+------+------| |------+------+------+------+------+------|
+ * | Tab  | CLAP | CUM  | BNIS | BUTT | CAR  | | FIRE | REDB | MONY | 100  | SOS | Entr |
+ * |------+------+------+------+------+------| |------+------+------+------+------+------|
+ * |      | CELE | PRAY | NAIL |  OK  | THNK | | UNAM | HEYE | COOL | EYES | SMIR | Del  |
+ * |------+------+------+------+------+------| |------+------+------+------+------+------|
+ * | Shft | TRIU | SCRM | VOMI | DTIV | EXPL | | HAIR | DANC | STRN | LEFT | RGHT | Shft |
+ * |------+------+------+------+------+------| |------+------+------+------+------+------|
+ * | Ctrl |      | Alt  | GUI  |      |      | |      |      | GUI  | Alt  |      | Ctrl |
+ * `-----------------------------------------' `-----------------------------------------'
+ */
+
+[_EMOJI] = LAYOUT( \
+  TO(0)   , XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX , /**/  RESET   ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   XXXXXXX ,   _______ , \
+  _______ , X(CLAP) ,   X(CUM)  ,   X(BNIS) ,   X(BUTT) ,   X(CAR)  , /**/  X(FIRE) ,   X(REDB) ,   X(MONY) ,   X(HNDR) ,   X(SOS)  ,   _______ , \
+  XXXXXXX , X(CELE) ,   X(PRAY) ,   X(NAIL) ,   X(OK)   ,   X(THNK) , /**/  X(UNAM) ,   X(HEYE) ,   X(COOL) ,   X(EYES) ,   X(SMIR) ,   KC_DEL  , \
+  _______ , X(TRIU) ,   X(SCRM) ,   X(VOMI) ,   X(DTIV) ,   X(EXPL) , /**/  X(HAIR) ,   X(DANC) ,   X(STRN) ,   X(LEFT)   ,   X(RGHT) ,   _______ , \
+  _______ , _______ ,   _______ ,   _______ ,   _______ ,   XXXXXXX , /**/  XXXXXXX ,   _______ ,   _______ ,   _______ ,   XXXXXXX ,   _______  \
+),
+
+};
+
+void matrix_init_user(void) {
+  _delay_ms(20); // Gets rid of tick
+  set_unicode_input_mode(UC_LNX);
```

Comment:
What do you mean 'only set it once'? Should this be defined somewhere else/in a different fashion?

Reply:
Nope.  This function actually writes to EEPROM, so it should be loaded automatically on future loads. In theory.

Reply:
Okay, so what I got from this, was:
- get rid of the `matrix_init_user()` function since it's already set and doing this would set it twice
- use `UC_LNX(unicode_map_index)` to send unicode strings up to 0xFFFF (which the defined emojis fall under)

Am I correct in this, or just completely missing what you're saying? :)

Reply:
So the above did not work since it wouldn't compile.  
I just removed the delay like you said, but no clue how I can correctly input these unicodes otherwise, so no real changes there.  
If I'm missing something from the docs, please do explain because I'm not seeing it :)